### PR TITLE
Documentation for `uniq`

### LINKED ISSUE #311: Create reference for uniq

### PR DESCRIPTION
**Pull request by ehuang6**
# Advanced HW 5 submission

Closes #311 

Name: Eric Huang
Uniqname: elhuang

### CODE REVIEW

**Comment by mterwill** [comment]
@ehuang6: please follow up by this Friday or we'll close the PR and won't be able to give you credit for the assignment.

**Comment by ehuang6** [comment]
Updated to link to sort page and added examples for uniq -c flag. I pushed to my own repo, do I need to open another pull request?

File: _commands/basics/uniq.md [hunk 1]
```diff
@@ -87,6 +107,33 @@ $ uniq -D numbers.txt
...
+2
+2
+0
+2
+1
+1
+3
+4
+4
+$ uniq -D numbers.txt
```
Comment (mterwill) [inline _commands/basics/uniq.md]: I think you forgot to change the flag here.

File: _commands/basics/uniq.md [hunk 2]
```diff
@@ -87,6 +107,33 @@ $ uniq -D numbers.txt
...
+1 0
+1 2
+2 1
+1 3
+2 4
+~~~
+
+##### Break it down
+
+ * The `-c` option tells `uniq` to print 2 all unique lines, retaining the first line of consecutive repeated lines, and also generate a count of the number of consecutive repeated lines. For unique lines, the count will be 1.
```
Comment (mterwill) [inline _commands/basics/uniq.md]: The wording "to print 2" doesn't make sense to me

File: _commands/basics/uniq.md [hunk 3]
```diff
@@ -36,6 +36,26 @@ _Notice that the 2 that follows the 3 is not filtered, even though it is repeate
...
+2
+2
+0
+2
+1
+1
+3
+4
+4
+$ numbers.txt | sort | uniq -u
```
Comment (mterwill) [inline _commands/basics/uniq.md]: Isn't `uniq -u` just `uniq`? If so, don't make it more confusing than it has to be.

**Comment by ehuang6** [comment]
@mterwill Updated to fix the typos

File: _commands/basics/uniq.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+2
+2
+0
+2
+1
+1
+3
+4
+4
+$ numbers.txt | sort | uniq
```
Comment (mterwill) [inline _commands/basics/uniq.md]: This should be `cat numbers.txt` or `sort numbers.txt`

Comment (ehuang6) [inline _commands/basics/uniq.md]: Fixed

**Comment by mterwill** [comment]
This looks great. Thanks for working through the changes.

**Pull request merged**